crates/typed-store/src/metrics.rs

```diff
@@ -33,8 +33,8 @@ pub struct SamplingInterval {
 
 impl Default for SamplingInterval {
     fn default() -> Self {
-        // Disabled by default
-        SamplingInterval::new(Duration::ZERO, u64::MAX - 1)
+        // Enabled with 10 second interval
+        SamplingInterval::new(Duration::from_secs(10), u64::MAX - 1)
```

Comment:
I just added metrics that were being sampled at 5s interval and it caused alot of spam, was suggested to switch to 60s interval by Allan. Might be worth checking in with him